Firestore/core/src/firebase/firestore/immutable/sorted_set.h

```diff
@@ -17,12 +17,14 @@
 #ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_IMMUTABLE_SORTED_SET_H_
 #define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_IMMUTABLE_SORTED_SET_H_
 
+#include <algorithm>
 #include <utility>
 
 #include "Firestore/core/src/firebase/firestore/immutable/sorted_map.h"
 #include "Firestore/core/src/firebase/firestore/immutable/sorted_map_base.h"
 #include "Firestore/core/src/firebase/firestore/util/comparison.h"
 #include "Firestore/core/src/firebase/firestore/util/firebase_assert.h"
+#include "Firestore/core/src/firebase/firestore/util/hashing.h"
```

Comment:
Is this being used? ... Oh, I see; you're using it in the test.

Are you trying to avoid defining std::hash(SortedSet)? i.e.

```
template<...>
struct std::hash<SortedSet<...>> {
  size_t operator()(SortedSet<...> s) { return util::Hash(s); }
}
```